koku/masu/processor/parquet/parquet_report_processor.py

```diff
@@ -475,7 +484,12 @@ def convert_csv_to_parquet(self, csv_filename):  # noqa: C901
             col_names = pd.read_csv(csv_filename, nrows=0, **kwargs).columns
             if self.ingress_reports:
                 REQUIRED_COLS = set(CSV_REQUIRED_COLUMNS.get(self._provider_type))
+                missing_cols = False
                 if not set(col_names).issuperset(REQUIRED_COLS):
+                    missing_cols = True
+                    if missing_cols and self._provider_type in {Provider.PROVIDER_AWS, Provider.PROVIDER_AWS_LOCAL}:
```

Comment:
Since the return value for `self.col_checker` is None if its not one we have defined a provider for, we could probably just use that as the second condition for this.
```suggestion
                    if missing_cols and self.col_checker:
```